### DISCUSSION TITLE
Small help for where I struggled initially

### DISCUSSION THREAD

**Discussion by Sethz** [krojew/OpenRGB-Ambient#2] [Show and tell]
As far as setup goes:

In OpenRGB in the Ambient Tab you got two sliders for each side of the stripes.

Top, Bottom, Right, Left

To set it up one needs to check preview and then use the two sliders on the channel that only this channel is lit up (as soon as you move a slider on a channel it updates for that channel only - so move the Top sliders until only top is light then move the Bottom sliders until only bottom is lit and so forth) after that uncheck preview and you should be good to go.

(Don't forget to check the device in the ambient tab).

**Comment by krojew** [comment]
Thanks for writing the instructions! I should have added them to the readme file in the first place. Can I paste your instructions there?

**Comment by darmach** [comment]
I think I'm a bit confused - I'm trying to use that with my led strip connected to ARGB header of my motherboard - like the default Effects library Ambient plugin. I don't think I can use that this way... Can I? (Looking for alternatives for default Effects library Ambient effect - as it lacks color (temp.) correction

**Reply by krojew** [reply]
Hard to say - I never tested it in such configuration. Try and see how it goes.
BTW. the effects plugin you tried uses a very inefficient way to set the colors, so I would advise against using it.
